### DISCUSSION TITLE
枪在手，跟我走？

### DISCUSSION THREAD

**Discussion by 306732407** [WeNeedHome/SummaryOfLoanSuspension#374] [General]
![c3b28b727d867cabbeb6f923266360a](https://user-images.githubusercontent.com/17486996/178942156-684350c1-3125-437e-8711-2aba0de3d888.jpg)

**Comment by JiabaoZhu** [comment]
想给你点赞，又怕你枪毙的时候血溅我脸上

**Comment by Plutus2Daniel** [comment]
看来微信QQ等群，需要加强监管

**Comment by 442FSJ** [comment]
去碉楼，拿回你们自己的东西

**Comment by hhyyy9** [comment]
开门，查水表

**Comment by jaysen365** [comment]
天从来都是黑的
